ci/run_cudf_polars_pytests.sh

```diff
@@ -12,7 +12,7 @@ cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../python/cudf_polars/
 python -m pytest --cache-clear "$@" tests --executor in-memory
 
 # Test the default "streaming" executor
-python -m pytest --cache-clear "$@" tests
+python -m pytest --cache-clear "$@" tests --executor streaming
```

Comment:
The change to `conftest.py` makes this change unnecessary. However, I prefer making the command more explicit anyway.